tests/CMakeLists.txt

```diff
@@ -9,7 +9,24 @@ if(NOT Python3_FOUND)
   message(FATAL_ERROR "Python3 support NOT FOUND! Python3 is needed to compare the solutions for the TGV case.")
 endif()
 
-# TGV is from here
+# We need to find a way to check if 2DECOMP&FFT is build with ADIOS2
+option(USE_ADIOS2 "Build XCompact with ADIOS2 library" OFF)
+
 # TGV is the only test active by default
 add_subdirectory(TGV-Taylor-Green-vortex)
+if (${BUILD_TESTING_FULL})
+  add_subdirectory(ABL-Atmospheric-Boundary-Layer)
+  add_subdirectory(Cavity)
+  add_subdirectory(Channel)
+  if(NOT ${USE_ADIOS2})
```

Comment:
What is the connection between ADIOS2 IO and testing the Cylinder case ?

Reply:
The support for ADIOS2 and Cylinder is not fully there. For this reason we avoid the case